pages/basic_example.md

```diff
@@ -1,5 +1,5 @@
 ---
-layout: page
+layout: otherpage
 title: Basic example
 order: 1
```

Comment:
If you turn it into otherpage you can remove `order` as this is only used the sort the menu entries.